tpl/.drone.yml.gotmpl

```diff
@@ -32,6 +39,13 @@ steps:
 {{- else if eq .provider "msteams" }}
   - name: notify-start
     image: diyfr/drone_plugin_teams
+    resources:
+      limits:
+        cpu: 200
+        memory: 200Mi
+      requests:
+        cpu: 100
+        memory: 100Mi
```

Comment:
Can we make this a repeatable block? That we can call once - in a variable, function or something similar? 
Since all the resources are the same. Would make maintaining in the long run easier, I think

Reply:
I tend to stray from abstracting similar resource blocks as those should usually be customized to the workload, so only the similar notification calls remain as suspect for your suggestion. Let me just do those